WordPress/Classes/ViewRelated/People/PeopleViewController.swift

```diff
@@ -87,44 +86,60 @@ public class PeopleViewController: UITableViewController, NSFetchedResultsContro
         if let personViewController = segue.destinationViewController as? PersonViewController,
             let selectedIndexPath = tableView.indexPathForSelectedRow
         {
-            personViewController.person = personAtIndexPath(selectedIndexPath)
             personViewController.blog = blog
+            personViewController.person = personAtIndexPath(selectedIndexPath)
         }
     }
 
 
     // MARK: - UIStateRestoring
 
-    override public func encodeRestorableStateWithCoder(coder: NSCoder) {
+    public override func encodeRestorableStateWithCoder(coder: NSCoder) {
         let objectString = blog?.objectID.URIRepresentation().absoluteString
         coder.encodeObject(objectString, forKey: RestorationKeys.blog)
         super.encodeRestorableStateWithCoder(coder)
     }
 
 
-    // MARK: - Helpers
+    // MARK: - Refresh Helpers
 
-    @IBAction func refresh() {
-        guard let blog = blog,
-            service = PeopleService(blog: blog) else {
-                return
+    @IBAction public func refresh() {
+        guard let blog = blog, service = PeopleService(blog: blog) else {
+            return
         }
+
         service.refreshTeam { [weak self] _ in
             self?.refreshControl?.endRefreshing()
+            self?.hideNoResultsIfNeeded()
         }
     }
 
+
+    // MARK: - Private Helpers
+
     private func personAtIndexPath(indexPath: NSIndexPath) -> Person {
         let managedPerson = resultsController.objectAtIndexPath(indexPath) as! ManagedPerson
         let person = Person(managedPerson: managedPerson)
         return person
     }
 
+    private func displayNoResultsIfNeeded() {
+        if resultsController.fetchedObjects?.count > 0 {
+            return
+        }
+
+        noResultsView.titleText = NSLocalizedString("Loading...", comment: "")
```

Comment:
Yup, this is actually displayed when the cache is empty, and a backend call is taking place.

Thanks for checking!